.circleci/config.yml

```diff
@@ -70,6 +75,18 @@ jobs:
           key: v1-jackdaw-{{ .Branch }}-{{ .Revision }}
           paths:
             - .
+  checkout_tags:
+    <<: *build_config
+    steps:
+      - *restore_repo
+      - run: git config --global url."https://github.com/".insteadOf "git@github.com:"
```

Comment:
This little dance is required because
  a. the cached repo doesn't include the tags so the subsequent `checkout` step fails.
  b. the clone url for the repo uses `git:github...` so the fetch triggers an interactive ssh host check so we want to switch to using https